rust-connectors/sinks/kafka/src/main.rs

```diff
@@ -1,13 +1,21 @@
 use fluvio_connectors_common::git_hash_version;
-use fluvio_connectors_common::opt::{CommonConnectorOpt, Record};
+use fluvio_connectors_common::opt::CommonConnectorOpt;
 use fluvio_future::tracing::{debug, info};
```

Comment:
```suggestion
use fluvio_future::tracing::{debug, info, error};
```

This for https://github.com/infinyon/fluvio-connectors/pull/306/files#r927820467